### PR TITLE
TA-36: added missing code from merge conflict

### PR DESCRIPTION
**Pull request by millifrill**

### CODE REVIEW

File: src/test/java/com/example/produktapi/controller/ProductControllerTest.java [hunk 1]
```diff
@@ -5,9 +5,9 @@
 import static org.hamcrest.Matchers.hasItems;
 
 import io.restassured.RestAssured;
-import io.restassured.response.Response;
-import org.junit.jupiter.api.Assertions;
+import org.apache.coyote.Response;
```
Comment (NicklasC) [inline src/test/java/com/example/produktapi/controller/ProductControllerTest.java:8]: import io.restassured.response.Response;

**Review by NicklasC — CHANGES_REQUESTED** [review]
It seems import io.restassured.response.Response has been replaced with import org.apache.coyote.Response. This did not compile successfully.

**Comment by millifrill** [comment]
image

**Review by NicklasC — APPROVED** [review]
Now it looks good! Good job.

**Pull request merged**